modules/using-certificates-with-workbenches.adoc

```diff
@@ -0,0 +1,24 @@
+:_module-type: PROCEDURE
+
+[id='using-certificates-with-workbenches_{context}']
+= Using certificates with workbenches
+
+[IMPORTANT]
+====
+ifdef::upstream[]
+Self-signed certificates apply to workbenches that you create after configuring self-signed certificates centrally as described in link:{odhdocshome}/installing-open-data-hub/#understanding-certificates_certs[Understanding certificates in {productname-short}]. There is no change to workbenches that you created before configuring self-signed certificates.
+endif::[]
+ifndef::upstream[]
+ifdef::cloud-service[]
+Self-signed certificates apply to workbenches that you create after configuring self-signed certificates centrally as described in link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}/working-with-certificates_certs[Working with certificates]. There is no change to workbenches that you created before configuring self-signed certificates.
+endif::[]
+ifdef::self-managed[]
+Self-signed certificates apply to workbenches that you create after configuring self-signed certificates centrally as described in link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}/working-with-certificates_certs[Working with certificates] (for disconnected environments, see link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}_in_a_disconnected_environment/working-with-certificates_certs[Working with certificates]. There is no change to workbenches that you created before configuring self-signed certificates.
+endif::[]
+endif::[]
+====
+
+== Creating data science pipelines with Elyra and self-signed certificates
```

Comment:
Is this in both modules intentionally?

Reply:
It is - I don't love the repetition, but it's relevant for both IDE and pipelines. Could remove from Workbenches if we think it'll be an issue?